src/index.html

```diff
@@ -11,8 +11,50 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:300,regular,500,700,900"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__img-cont">
+        <img
+          src="./images/imac.jpeg"
+          alt="product-img"
+          class="card__img"
+        />
+      </div>
+      <div class="card__product-title-cont">
```

Comment:
Let's make the name of the class more meaningful. Because `cont` is not clear. Try something like card__product-main-info